tethys_gizmos/gizmo_options/map_view.py

```diff
@@ -313,7 +313,7 @@ class MapView(TethysGizmoOptions):
     """  # noqa: E501
     gizmo_name = "map_view"
     ol_version = '5.3.0'
-    cdn = 'https://cdn.jsdelivr.net/npm/openlayers@{version}/dist/ol{debug}.{ext}'
+    cdn = 'https://cdn.jsdelivr.net/gh/openlayers/openlayers.github.io@master/en/v{version}/{folder}/ol{debug}.{ext}'
```

Comment:
Are there other options you would pass in for `folder`? It seems to me that we can do without `folder` being a variable and just hard-code it to be "build" in the URL.

Reply:
For JavaScript, this folder is "build", but for CSS it is "css". See lines below.

Reply:
Ah, I see now. Sorry, I didn't look closely enough I suppose.